src/libasr/codegen/asr_to_wasm.cpp

```diff
@@ -2105,9 +2105,9 @@ class ASRToWASMVisitor : public ASR::BaseVisitor<ASRToWASMVisitor> {
 
     void visit_Assert(const ASR::Assert_t &x) {
         this->visit_expr(*x.m_test);
-        wasm::emit_i32_eqz(m_code_section, m_al);
         wasm::emit_b8(m_code_section, m_al, 0x04);  // emit if start
         wasm::emit_b8(m_code_section, m_al, 0x40);  // empty block type
+        wasm::emit_b8(m_code_section, m_al, 0x05);  // starting of else
         if (x.m_msg) {
             std::string msg =
                 ASR::down_cast<ASR::StringConstant_t>(x.m_msg)->m_s;
```

Comment:
The previous version was more correct, right? The previous version would say -> If `Assert` condition failed, raise an error (which is what `assert` does, I think).

Reply:
The main motive for these changes was that `visit_I32Eqz()` in the `wasm_to_x86` is not working correctly.
Here, the changes implies:
```py
If True:
	// Do nothing
else:
	print("AssertionError")
```
which is similar to `assert False`

Reply:
The same approach as we do in the LLVM backend: https://github.com/lcompilers/lpython/blob/d4a6d5360ca520d9a25d658e6660cb39a2eda4e2/src/libasr/codegen/asr_to_llvm.cpp#L4845-L4862